Classes/Notifications/NewFeaturesController.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  NewFeaturesController.swift
+//  Freetime
+//
+//  Created by Ryan Nystrom on 11/22/18.
+//  Copyright © 2018 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+import GitHubAPI
+import IGListKit
+
+final class NewFeaturesController {
+
+    var version: String {
+        return Bundle.main.infoDictionary?["CFBundleShortVersionString"] as? String ?? ""
```

Comment:
Pretty sure this is safe; maybe use implicitly unwrapped optionals or an `assert`?

Reply:
An `assert` would be perfect then?